Examples/CliClient/Program.cs

```diff
@@ -340,30 +378,23 @@ async Task<Connection> CreateUpgradeConnection(string host, int port)
             .SetHttp2Settings(config.Settings)
             .Build();
 
-        // Create a TCP connection
-        logger.LogInformation($"Starting to connect to {host}:{port}");
-        var tcpClient = new TcpClient();
-        await tcpClient.ConnectAsync(host, port);
-        tcpClient.Client.NoDelay = true;
-        logger.LogInformation("Connected to remote");
-
-        // Create HTTP/2 stream abstraction on top of the socket
-        var wrappedStreams = tcpClient.Client.CreateStreams();
-        var upgradeReadStream = new UpgradeReadStream(wrappedStreams.ReadableStream);
+        var (readableStream, writeableStream) = await CreateStreams(scheme, host, port);
+        var upgradeReadStream = new UpgradeReadStream(readableStream);
 
         var needExplicitStreamClose = true;
         try
         {
+            var upgradeValue = scheme == "https" ? "h2" : "h2c";
```

Comment:
This can always be `h2c` due to the other change.
```suggestion
            // Upgrades are only possible for non-encrypted connections.
            var upgradeValue = "h2c";
```

Reply:
Can you please click apply the suggestion? That's not possible for me.
I will merge it then.
Thanks again!